spec/pluralrules.html

```diff
@@ -102,12 +102,19 @@ <h1>GetOperands ( _s_ )</h1>
        </emu-table>
     </emu-clause>
 
-    <emu-clause id="sec-pluralruleselect" aoid="PluralRuleSelect">
-      <h1>PluralRuleSelect ( _locale_, _type_, _n_, _operands_ )</h1>
-
-      <p>
-        When the PluralRuleSelect abstract operation is called with four arguments, it performs an implementation-dependent algorithm to map _n_ to the appropriate plural representation of the Plural Rules Operands Record _operands_ by selecting the rules denoted by _type_ for the corresponding _locale_, or the String value *"other"*.
-      </p>
+    <emu-clause id="sec-pluralruleselect" type="implementation-defined abstract operation">
```

Comment:
The `aoid` is gone, I suppose automagic linking of this AO from algorithms won't work anymore.

Reply:
`aoid` is now pulled from the structured header, cf. https://github.com/tc39/ecmarkup/blob/6253fe32dec8a8b7f4271e76f734fbab2ee4c5f4/src/Clause.ts#L122-L143

Reply:
oh wow, this is great. I should really keep up with all the exciting new stuff in ecmarkup